src/reconcile/rec_child.c

```diff
@@ -252,6 +252,12 @@ __wti_rec_child_modify(
              */
             if (!WT_REF_CAS_STATE(session, ref, WT_REF_DELETED, WT_REF_LOCKED))
                 break;
+
+            /* TODO: support delta for fast truncate. */
+            if (build_delta != NULL) {
+                *build_delta = false;
+                r->delta.size = 0;
+            }
```

Comment:
so in this case since we are reconciling the parent page, and we find that the child_ref is deleted, then the only way this is possible is fast truncate?

Reply:
It's also possible for checkpoint cleanup. But we don't have a way to distinguish them. We will disable them together here for now.

Reply:
cool